app/src/main/java/dev/spikeysanju/expensetracker/view/details/TransactionDetailsFragment.kt

```diff
@@ -126,19 +128,26 @@ class TransactionDetailsFragment : BaseFragment<FragmentTransactionDetailsBindin
             return
         }
 
-        val imageURI = binding.transactionDetails.detailView.drawToBitmap().let { bitmap ->
-            saveBitmap(requireActivity(), bitmap)
-        } ?: run {
-            toast("Error occurred!")
-            return
-        }
+        // unHide the app logo and name
+        binding.transactionDetails.appIconForShare.show().let {
+            binding.transactionDetails.appNameForShare.show().let {
+                val imageURI = binding.transactionDetails.detailView.drawToBitmap().let { bitmap ->
+                    binding.transactionDetails.appIconForShare.hide()
+                    binding.transactionDetails.appNameForShare.hide()
+                    saveBitmap(requireActivity(), bitmap)
+                } ?: run {
+                    toast("Error occurred!")
+                    return
+                }
```

Comment:
@brookmg  This is looks good. Nested scope functions will cause readability issue. Shall we try this way? 👇

<img width="1134" alt="Screenshot 1942-12-09 at 23 10 24" src="https://user-images.githubusercontent.com/23400022/109429131-4039bd00-7a20-11eb-8ba3-4a3582e25b7d.png">

Reply:
Sure. This is a much better way to do it.